pkg/grpc/grpc.go

```diff
@@ -112,6 +112,15 @@ func (g *Manager) GetGRPCConnection(address, id string, namespace string, skipTL
 	defer cancel()
 
 	dialPrefix := GetDialAddressPrefix(g.mode)
+	if sslEnabled {
+		dialPrefix = "https://"
```

Comment:
for Kubernetes also does it need to change to https?

Reply:
Yes, this is going to the app, not to another Dapr instance.